pkg/ddc/jindo/types.go

```diff
@@ -1,3 +1,18 @@
+/*
+Copyright <year> The <project-name> Author.
```

Comment:
This should be `Copyright 2022 The Fluid Authors.`

Reply:
> This should be `Copyright 2022 The Fluid Authors.`

I have changed it and commit it.